geppetto-ui/src/components/big-image-viewer/showcase/BigImageViewerConfig.js

```diff
@@ -0,0 +1,18 @@
+import BigImageViewerExample
+    from "./examples/BigImageViewerExample";
+
+
+export const BigImageViewerConfig = {
+    'name': "BigImageViewer Main Title",
+    'description': "BigImageViewer Main Description",
```

Comment:
Idea for the future: get the description from the readme file inside the component folder